bindata/network/additional-networks/sriov/sriov-cni.yaml

```diff
@@ -0,0 +1,42 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: sriov-cni
+  namespace: openshift-sriov
+  annotations:
+    kubernetes.io/description: |
+      This daemonset copies the SR-IOV CNI plugin on to each node.
+spec:
+  selector:
+    matchLabels:
+      app: sriov-cni
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sriov-cni
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      hostNetwork: true
```

Comment:
why do we need hostNetwork for a container that merely copies a file to host directory?

Reply:
this is not necessary, will remove